src/cmds/azure-devops.ts

```diff
@@ -0,0 +1,96 @@
+import * as debugLib from "debug";
+import { AzureDevopsTarget, ContributorMap } from "../lib/types";
+import { SCMHandlerClass } from "../lib/common/SCMHandler";
+import { SourceType } from "../lib/snyk";
+import { fetchAzureDevopsContributors } from "../lib/azure-devops/azure-devops-contributors";
+
+
+const debug = debugLib("snyk:azure-devops-count");
+const azureDefaultUrl = "https://dev.azure.com/";
+
+const d = new Date();
+export const threeMonthsDate = d.getFullYear() + "/" + (d.getMonth() - 2) + "/" + d.getDate() + " " + d.getHours() + ":" + d.getMinutes();
+
+export const command = ["azure-devops"];
+export const desc = "Count contributors for azure-devops.\n";
+
+export const builder = {
+// options like
+  token: { required: true, default: undefined, desc: "Azure Devops token" },
+  org: {
+    required: true,
+    default: undefined,
+    desc: "Your Org name in Azure Devops e.g. https://dev.azure.com/{OrgName}",
+  },
+  projectKeys: {
+    required: false,
```

Comment:
looking at the code, not sure I see a situation where not having the projectKey is actually ok. If it is require, then I'd suggest to make it required here.

Reply:
It was left there from when I used the azure lib that could get you anything just by supplying the Orgname.
Will turn to required.